lib/tasks/import.rake

```diff
@@ -15,6 +15,7 @@ namespace :import do
 
     documents_to_import = WhitehallMigration::DocumentImport.where(whitehall_migration_id: whitehall_migration.id).count
     puts "Identified #{documents_to_import} documents to import"
+    puts "#{Plek.new.external_url_for("content-publisher")}/whitehall-migration/#{whitehall_migration.id}"
```

Comment:
Cool! - It'll be neat having a link to click after starting a migration.

We can use the Rails route helpers rather than writing our own route. This can be done by importing Rails.application.routes.url_helpers and then using a method like `whitehall_migration_url(whitehall_migration, host: Plek.new.external_url_for("content-publisher"))`